packages/domain/lib/src/services/analytics_service.dart

```diff
@@ -0,0 +1,3 @@
+abstract class FirebaseAnalyticsService{
```

Comment:
FirebaseAnalyticsService doesn't seem to be smth abstract. name it plz AnalyticsService